src/many-migration/tests/migrations.rs

```diff
@@ -1,465 +0,0 @@
-#![feature(used_with_arg)] // Required to build the test with Bazel
```

Comment:
Why did you remove those tests? Where are those tested now?

Reply:
My mistake, I deleted the wrong files.